cmd/mysql/stream_push.go

```diff
@@ -12,12 +13,14 @@ const StreamPushShortDescription = ""
 
 // streamPushCmd represents the streamPush command
 var streamPushCmd = &cobra.Command{
-	Use:   "stream-push",
+	Use:   "stream-push command\\ [command\\ args]",
```

Comment:
It seems to be unextendable. Also, it will harden flags addition for "stream-push" command. Let's move inner command to env variable